sale_partner_source/models/res_partner.py

```diff
@@ -0,0 +1,9 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    source_id = fields.Many2one('utm.source', string='Source')
```

Comment:
Why not putting this outside of a 'sale` module to keep the utm source of the partner without having to install sale module ?

Reply:
Ys, I understood that part. What I want to say is a module that stores the utm source on partner level could be great without the sale feature

Reply:
Oh, I understand your point now. I'll make adjustments